CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.24
+
+_Feb 16, 2023_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Hungarian (hu-HU) locale
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.17.24` / `@mui/x-data-grid-pro@v5.17.24` / `@mui/x-data-grid-premium@v5.17.24`
+
+#### Changes
+
+- [DataGrid] Allow to pass props to the `TrapFocus` inside the panel wrapper (#7897) @m4theushw
+- [DataGrid] Avoid unnecessary rerenders after `updateRows` (#7945) @cherniavskii
+- [DataGrid] Fix virtual scroller not updating layout after pinning the column (#7953) @cherniavskii
+- [DataGridPro] Change cursor when dragging column (#7878) @m4theushw
```

Comment:
```suggestion
- [DataGridPro] Change cursor when dragging the column (#7878) @m4theushw
```

Reply:
Probably this would be more correct grammatically, wouldn't it? 🤔 
```suggestion
- [DataGridPro] Change cursor when dragging a column (#7878) @m4theushw
```

Reply:
Right!